videos-list/22-Nikoleta.md

```diff
@@ -0,0 +1,32 @@
+```txt
+
+Speaker: Nikoleta Spinu
```

Comment:
The filename and speaker name here are not correct. It should be Nicoleta Spînu, Nicoleta with a "c" and the "i" on the surname with a circumflex accent. The bio looks right and the current youtube description is correct too.

Also, related but different issue the youtube thumbnail seems to have a "c" (correct) but no circumflex accent. cc @reshamas